Directory.Build.props

```diff
@@ -43,9 +43,9 @@
     <PackageReference Include="Asyncify" Version="0.9.7" PrivateAssets="all" />
     <PackageReference Include="Meziantou.Analyzer" Version="1.0.646" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3" PrivateAssets="All" />
-    <PackageReference Include="SecurityCodeScan" Version="3.5.3" PrivateAssets="all" />
+    <PackageReference Include="SecurityCodeScan.VS2019" Version="5.0.0" PrivateAssets="all" />
```

Comment:
Cool with updated analyzer - don't know it yet - but if so, shouldnt we add it to coding-rules setup - so we are aligned?

Reply:
This is the same, the analyzer has been renamed and version set for 5.0

Reply:
I updated `SecurityCodeScan` to `SecurityCodeScan.VS2019` due to a warning that the former was deprecated.

It makes sense to update coding-rules, but I am not sure what the update frequency of this is.